spec/models/canonical/material_spec.rb

```diff
@@ -86,6 +86,30 @@
     end
   end
 
+  describe 'scopes' do
+    describe '::with_craftable' do
+      subject(:with_craftable) { described_class.with_craftable }
+
+      let!(:included_models) { create_list(:canonical_material, 2, :with_craftable) }
+      let!(:excluded_model) { create(:canonical_material, :with_temperable) }
```

Comment:
This should go in a `before` block instead of using a memoised helper, since it isn't invoked in the spec.